articles/analytics/integrations/google-analytics/index.md

```diff
@@ -0,0 +1,57 @@
+---
+description: This article explains how to install and configure the Google Analytics for Auth0 integration.
+---
+
+# Google Analytics for Auth0
+
+This article explains how to install and configure the Google Analytics for Auth0 integration. You can use this integration on your own page that is using Lock or you can use this on the hosted Lock pages. Additionally, you will find instructions on how to configure funnels and reports inside of Google Analytics to get the most out of this integration.
+
+
+## Install
+In order to add the integration to your app you simply need to reference the Auth0 Analytics.js script on any pages in your app that have Auth0 Lock. The script must be refernced after Lock and must have the configuration options set before the script reference.
+
+```
+<script>
+window.auth0AnalyticsOptions = {
+  // Options
+}
+</script>
+<script src="https://cdn.auth0.com/js/lock/10.x/lock.min.js"></script>
+<script src="https://cdn.auth0.com/js/analytics/X.Y.Z/analytics.min.js"></script>
+```
+
+> Notice: The script version above uses a placeholder version `X.Y.Z`. In order to determine the latest release view the [releases in github](https://github.com/auth0/auth0-analytics.js/releases/). For example, to reference release 1.2.0 use `https://cdn.auth0.com/js/analytics/1.2.0/analytics.min.js`
```

Comment:
@ntotten Can you change this to the format I used in the facebook doc?

```
::: note
The script version above uses a placeholder version `X.Y.Z`. In order to determine the latest release view the [releases in github](https://github.com/auth0/auth0-analytics.js/releases/). For example, to reference release 1.2.0 use `https://cdn.auth0.com/js/analytics/1.2.0/analytics.min.js`
:::
```

For some reason I can't edit this doc using the online editor, the editor closes as soon as it loads.